e2e/cypress/integration/account_settings/general/focus_spec.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Account Settings > General', () => {
+    before(() => {
+        // # Go to Account Settings with "user-1"
+        cy.toAccountSettingsModal('user-1');
+    });
+
+    it('focus is in editing the full name', () => {
```

Comment:
The name of the test is the same across all the files. They should be different, right?

Reply:
since they are in different folders, I thought it would be ok. I'll change them